src/Pages/Dashboards.hs

```diff
@@ -798,153 +831,247 @@ renderDashboardListItem checked tmplClass title value description icon prview =
-
-    div_ [class_ $ "grid gap-5 " <> if dg.embedded then "grid-cols-1" else "grid-cols-2"] do
-      forM_ dg.dashboards \dashVM -> do
-        let dash = loadDashboardFromVM dashVM
-        let attrs =
-              if dg.embedded
-                then
-                  [ class_ "cursor-pointer"
-                  , hxPut_ ("/p/" <> dg.projectId.toText <> "/dashboards/" <> dashVM.id.toText)
-                  , hxVals_ "js:{...JSON.parse(document.getElementById(document.getElementById('dashboards-modal-widget-id').value + '_widgetEl').dataset.widget)}"
-                  , hxSwap_ "none"
-                  , hxExt_ "json-enc"
-                  ]
-                else [href_ ("/p/" <> dg.projectId.toText <> "/dashboards/" <> dashVM.id.toText)]
-        a_ ([class_ "rounded-xl border border-strokeWeak hover:border-strokeBrand-strong gap-3.5 p-4 bg-fillWeaker flex itemListItem group/i"] <> attrs) do
-          div_ [class_ "flex-1 space-y-2"] do
-            div_ [class_ "flex items-center gap-2"] do
-              span_ [class_ "group-hover/i:underline underline-offset-2"]
-                $ strong_ [class_ "font-medium"] (toHtml $ bool "Untitled" dashVM.title (dashVM.title /= ""))
-              span_ [class_ "leading-none", term "data-tippy-content" "This dashboard is currently your homepage."] do
-                when (isJust dashVM.homepageSince) $ faSprite_ "house" "regular" "w-4 h-4"
-            div_ [class_ "gap-2 flex items-center"] do
-              time_ [class_ "mr-2 text-textWeak", term "data-tippy-content" "Date of dashboard creation", datetime_ $ Utils.formatUTC dashVM.createdAt] $ toHtml $ formatTime defaultTimeLocale "%eth %b %Y" dashVM.createdAt
-              forM_ dashVM.tags (span_ [class_ "badge badge-neutral"] . toHtml @Text)
-          div_ [class_ "flex items-end justify-center gap-5"] do
-            button_ [class_ "leading-none", term "data-tippy-content" "click to star this dashboard"]
-              $ if isJust dashVM.starredSince
-                then faSprite_ "star" "solid" "w-5 h-5"
-                else faSprite_ "star" "regular" "w-5 h-5"
-            let widgetCount = maybe "0" (show . length . (.widgets)) dash
-            div_ [class_ "flex items-end gap-2", term "data-tippy-content" $ "There are " <> widgetCount <> " charts/widgets in this dashboard"] $ faSprite_ "chart-area" "regular" "w-5 h-5 text-iconNeutral" >> (span_ [class_ "leading-none"] . toHtml $ widgetCount)
-
-
-dashboardsGetH :: Projects.ProjectId -> Maybe Text -> ATAuthCtx (RespHeaders (PageCtx DashboardsGet))
-dashboardsGetH pid embeddedM = do
+          $ div_ [class_ "bg-fillBrand-strong px-2 py-4 rounded-xl w-full flex items-center"]
+          $ img_ [src_ "/public/assets/svgs/screens/dashboard_blank.svg", class_ "w-full rounded overflow-hidden", id_ "dItemPreview"]
+        let teamList = decodeUtf8 $ AE.encode $ (\x -> AE.object ["name" AE..= x.handle, "value" AE..= x.id]) <$> dg.teams
```

Comment:
Our tag handling system should be unified. We have this code here and in src/Pages/LogExplorer/Log.hs we have almost the exact js function. I expect that we create a component or reusable way of having that functionality without introducing so much new code.